src/psyclone/psyir/nodes/node.py

```diff
@@ -1076,6 +1077,8 @@ class or a tuple of classes) is found. This can be used to avoid
         :type my_type: type | Tuple[type, ...]
         :param stop_type: class(es) at which recursion is halted (optional).
         :type stop_type: Optional[type | Tuple[type, ...]]
+        :param depth: the depth value the instances must have (optional).
+        :type depth: Optional[type | int]
```

Comment:
Should just be `Optional[int]` I think?

Reply:
Thanks, addressed this in 579d09517ba612522dc45e3f0e01c167e160f280.